docs/customizing.rst

```diff
@@ -52,34 +52,34 @@ they are created or updated.
 
 .. code-block:: python
 
-    from datetime import datetime
+    from datetime import datetime, timezone
     from sqlalchemy import DateTime, Integer, String
     from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column, declared_attr
 
     class TimestampModel(db.Model):
         __abstract__ = True
-        created: Mapped[datetime] = mapped_column(DateTime, nullable=False, default=datetime.utcnow)
-        updated: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+        created: Mapped[datetime] = mapped_column(default=lambda: datetime.now(timezone.utc))
```

Comment:
Have you run this code?

Reply:
Yep, runs successfully in a minimal app.